tests/odm/conftest.py

```diff
@@ -197,118 +297,38 @@ async def session(cli):
     await s.end_session()
 
 
+@pytest.fixture()
+async def deprecated_init_beanie(db):
```

Comment:
i wanted to explicitly separate deprecated usages from not deprecated ones
i also added assertions those deprecations are actualy raised